src/gpodder/gtkui/shownotes.py

```diff
@@ -128,6 +128,9 @@ def hide_pane(self):
         self.pane_is_visible = False
         self.shownotes_pane.hide()
 
+    def on_link_clicked(self, url):
+        util.open_website(url)
+
```

Comment:
Is there any value in having this alias to `open_website`? If you remove it, then this PR would be a one-liner.

Reply:
Ah, you already commented on this. Either way is fine for me...

Reply:
Why is on_link_clicked() stuffed in between pane-related methods?

Regarding the comment outside of this review, couldn't we add the function in the future if something actually needs it? Until then it only makes the code harder to follow. And the title for both text and html shownotes is still using open_website(), so some links are using it but not others.

The confirmation dialog was a good idea for anyone wanting to avoid accidental browser openings, or to see the link they are about to open. It could even be extended to always-open, ask, copy-only. There could be two settings, one for gpodder links and the other for foreign links. The open_website() function could get an optional parameter to pass the foreign link setting from shownotes and channel dialog. If no parameter is given, it would internally use the gpodder link setting. No indirection is required to implement this.

Reply:
Good point. PR updated, it's now a single-line change :)